homeworks/Vladimir.Karbivnychiy_VladimirSkarb/11-dependency-injection/car-list/src/app/directives/change-color-field.directive.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  Directive,
+  ElementRef,
+  AfterViewInit,
+} from '@angular/core';
+
+@Directive({selector: '[appChangeColorField]'})
+export class ChangeColorFieldDirective implements AfterViewInit {
+
+  constructor(private el: ElementRef) {}
+
+  ngAfterViewInit() {
+    const protocol: string = this.el.nativeElement.children[1].innerText;
```

Comment:
Sorry, didn't see this issue before.
This is not a safe way to get an element(
If you need to find a specific element, you can do it in a few solutions:
1) (best) pass the element as `@Input()` and use it from `this`;
2) (ok, but the first is best) use `querySelector` and `data-attributes`